src/DKGManager.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT AND Apache-2.0
+pragma solidity ^0.8.19;
+
+import {Ownable} from "solady/auth/Ownable.sol";
+import {DKG} from "./DKG.sol";
+import {IDKGMembership} from "./interfaces/IDKGMembership.sol";
+
+/// @title DKGManager
+/// @author Cryptonet
+/// @notice Contract to alert nodes of new DKGs
+contract DKGManager is Ownable {
+    /// @notice Emitted when a new DKG is registered
+    /// @param dkg The address of the deployed DKG
+    event NewDKGCreated(address dkg);
+
+    constructor(address owner) {
+        _initializeOwner(owner);
+    }
+
+    /// @notice Alerts nodes of new DKGs with by emitting an event
+    /// @dev Only the owner can register a new DKG
+    function registerNewDKG(DKG dkg) external onlyOwner {
```

Comment:
Is this the API we want ? 
Obviously if these contracts are upgradeable then fine, but I imagine we want more stg like
`registerNewDKG(address, type)` and nodes listening will interpret the type.
Now we are stuck with one concrete type of DKG contract.
Or am I missing stg ? Is upgradibility gonna solve this problem down the line ? (still not 100% sure of the impact, need ot read more about upgradability).

Reply:
So the typing here only helps with type safety from Solidity. There's no encoding of the types in the EVM/deployed bytecode. So we can actually send any address here and the contract will work.

You're right though, I can change this to `IDKG` to be more correct. And yes it's probably a good idea to add an identifier to IDKG, similar to `suite()` in IEncryptionOracle. What should this be called ... maybe a different word than `type()` since it can collide with keywords / common functions in other languages (rust, typescript, etc.)

Reply:
Yeah I think suite should work out fine because that's what we use elsewhere ?

Reply:
The key is not tied to any sort of encryption algorithm right?

So the values would be something like:
`enum Suite {BN254_KEYG1}`

Reply:
That's right. Then `enum Suite {BN254_KEYG1, BabyJubJub}`